phdcontrol.cpp

```diff
@@ -65,6 +65,7 @@ struct ControllerState
     bool saveSticky;
     int autoFindAttemptsRemaining;
     int waitSelectedRemaining;
+    bool dontUseStickyLock;

```

Comment:
can we invert this to be a positive attribute to reduce double negatives?
```suggestion
    bool useStickyLock;
```